packages/spor-react/src/layout/PressableCard.tsx

```diff
@@ -37,21 +39,20 @@ type PressableCardProps = BoxProps & {
  * </PressableCard>
  * ```
  *
- * For a static card with other color schemes, use the `StaticCard` component.
+ * For a static card with other color palette, use the `StaticCard` component.
  *
  * @see StaticCard
  */
 
-export const PressableCard = forwardRef<PressableCardProps, As>(
-  ({ children, variant = "floating", ...props }, ref) => {
-    const styles = useStyleConfig("PressableCard", {
-      variant,
-    });
-
+export const PressableCard = forwardRef<HTMLDivElement, PressableCardProps>(
+  ({ variant = "base", children, ...props }, ref) => {
+    const styles = useRecipe({ recipe: pressableCardRecipe });
```

Comment:
Tror kanskje variant må inn her, så foreslår at du skriver det slik:

```suggestion
    const recipe = useRecipe({ recipe: pressableCardRecipe });
    const styles = recipe({ variant });
```